ux.symfony.com/templates/components/Badge.html.twig

```diff
@@ -1,19 +1,34 @@
-<div {{ attributes.defaults({
-    class: 'Badge',
-}) }}>
-    <span class="Badge__label">
-        {%- if icon %}
-            <twig:ux:Icon name="{{ icon }}" class="Badge__icon" />
-        {% endif -%}
-        {{- label -}}
-    </span>
-    <span class="Badge__value">
-        {%- if url %}
-            <a href="{{ url }}" class="Badge__link">
-                {{- value -}}
-            </a>
-        {% else %}
-            {{ value }}
-        {% endif -%}
-    </span>
+{%- props variant = 'default', outline = false -%}
+{%- set style = html_cva(
+    base: 'inline-flex items-center rounded-full border px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2',
+    variants: {
+        variant: {
+            default: "border-transparent bg-primary text-primary-foreground hover:bg-primary/80",
+            secondary: "border-transparent bg-secondary text-secondary-foreground hover:bg-secondary/80",
+            destructive: "border-transparent bg-destructive text-destructive-foreground hover:bg-destructive/80",
+        },
```

Comment:
Please use a new dedicated folder for these components, but don't change existing ones used in the website :)